.gitattributes

```diff
@@ -1,2 +1,3 @@
 *.psd filter=lfs diff=lfs merge=lfs -text
 *.wav filter=lfs diff=lfs merge=lfs -text
+*.unity filter=lfs diff=lfs merge=lfs -text
```

Comment:
Why is .unity added to LFS?

Reply:
Mmm I think it's because my scene with the navmesh seemed to have reached 4.5 mb. What size should be under lfs?

Reply:
Only stuff bigger than 80-100mb. Regardless, don’t touch LFS before discussion with us as it is volatile and will corrupt existing file without proper migration on all of our local working copies.